modules/kubernetes-addons/velero/locals.tf

```diff
@@ -0,0 +1,56 @@
+locals {
+  name                 = "velero"
+  service_account_name = local.name
+  s3bucketprefix = "eks-tf-velero-backup"
+  s3bucketname = aws_s3_bucket.s3.id
+
+  default_helm_config = {
+    name        = local.name
+    chart       = local.name
+    repository  = "https://vmware-tanzu.github.io/helm-charts/"
+    version     = "2.23.6"
+    namespace   = local.name
+    description = "Velero AddOn Helm Chart"
+    values      = local.default_helm_values
+  }
+
+  default_helm_values = [templatefile("${path.module}/values.yaml", {
+    velero-sa-name = local.service_account_name
```

Comment:
Remove this from values.yaml as we use set_values for service-accounts anyway.

Reply:
Please remove this as you are using `set_values` anyway (which is the right way)